generators/python/src/fern_python/generators/pydantic_model/type_declaration_handler/discriminated_union/discriminated_union_with_utils_generator.py

```diff
@@ -1,9 +1,13 @@
+from re import A
```

Comment:
FER-2479: This can be greatly simplified if we respect the Pydantic version config